Patches/OpenCV/3.1.0/common.cmake

```diff
@@ -0,0 +1,150 @@
+# This file is included from a subdirectory
```

Comment:
Thanks, perfect. It should show up when I run gitk, etc. I don't know what happens in the github UI though, but works for my purposes.